### DISCUSSION TITLE
My app keeps crashing

### DISCUSSION THREAD

**Discussion by Opeyimika-sudo** [illacloud/illa-builder#3736] [Q&A] [answered]
Hi, 

I created an Illa cloud desktop app today. So far I built one page with the intention of oscillating between 2/3 tabs. The first tab was a Form which had a container. So the intention was that after submitting, it would go to View 2 of the Container. It does that but attempts to click the dropdown of the Container or simply, click anything on the page now, results in a display of 404, It seems that the page you were looking for could not be found and it gives me two buttons - Refresh and Back to Homepage.

https://github.com/illacloud/illa-builder/assets/54380080/a623990c-449a-4229-885e-cd2623ffe10b

Here's a [video attempt](https://drive.google.com/drive/folders/14rs452VSmsiluZT3Tf2fb3Ima8RNZEeb?usp=sharing) to reproduce the bug. I'd appreciate any help. I'm available to answer any following questions or to take anyone through the app so far in an attempt to solve the bug together.

Thanks, everyone. You have built a fantastic app with Illa cloud, I must say.

**Comment by AruSeito** [comment]
Heelo,Can u open the Chrome devTools? What can you see there when your app crashes?

**Reply by Opeyimika-sudo** [reply]
Hi @AruSeito,

Thank you for replying. Here's the screenshot of my Devtools.
![image](https://github.com/illacloud/illa-builder/assets/54380080/ba14a8ba-7da8-4a8f-9c87-7f6eb987ae46)

What makes it hard to debug/solve is that any clicking of a component causes the page to show the 404 error.

Thank you for your anticipated help.

**Reply by AruSeito** [reply]
This is bug and we will fix it as soon as possible

**Reply by AruSeito** [reply, marked as answer]
Hello, we have fixed.

**Comment by Opeyimika-sudo** [comment]
Thank you, AruSeito.
I’d check it out soon and revert.
